lib/internal_metric/global.rb

```diff
@@ -27,7 +27,8 @@ class Global < Base
               :missing_s3_uploads,
               :version_info,
               :readonly_sites,
-              :postgres_highest_sequence
+              :postgres_highest_sequence,
```

Comment:
I think we should just nuke this metric since the new metric effectively replaces this.